src/type/dto/webhook.dto.ts

```diff
@@ -33,6 +33,7 @@ export const InvoiceCreatedResponseSchema = z.object({
     status: z.nativeEnum(InvoiceStatus),
     total: z.number(),
     taxAmount: z.number().optional(),
+    taxPercentage: z.number().optional(),
```

Comment:
```suggestion
    taxPercentage: z.number().default(0),
```

Up to preference, but this will save us from having to check if `taxPercentage` is undefined and running ternary checks wherever this field is used.

In further calculation, anyNumber * 0 will be 0